utils/mil_tools/nodes/online_bagger.py

```diff
@@ -54,10 +54,35 @@ def get_params(self):
         """
         Retrieve parameters from param server.
         """
+        self.dir = rospy.get_param(rospy.get_name() + '/bag_package_path', default=os.environ['HOME'])
```

Comment:
This should probably default to a dated folder under the directory specified in the `BAG_DIR` environment variable if it exists.